utils/rpms/daos.spec

```diff
@@ -219,8 +219,10 @@ Requires: lbzip2
 Requires: attr
 %if (0%{?suse_version} >= 1315)
 Requires: lua-lmod
+Requires: libcapstone-devel
 %else
 Requires: Lmod
+Requires: capstone-devel
```

Comment:
Taking this to a new thread.  I think we are actually inconsistent here.   There are several other devel packages we would need to add here if this is the way we have done it.  I think we must have a script somewhere in which we install a bunch of devel packages in order to run the tests.  I think we should be consistent.

Reply:
I [would have] thought so too.  I even [said as much](https://daosio.atlassian.net/browse/DAOS-13476?focusedCommentId=115324) and went looking to see what the daos-client-tests RPM was requiring and was surprised myself how little it was.

But the only other place that we list any other packages to be installed during a test node provision is [here](https://github.com/daos-stack/daos/blob/master/ci/functional/required_packages.sh) which as I said earlier (is getting unwieldy and) I am trying to eliminate.

Ahhh.  But then there is https://github.com/daos-do/packer-ci-images/blob/06d13d571e7bb33fb73ca78ab2b7f0f484f28bc9/scripts/centos8/ci_packages.sh#L81-L139.  That is the list of packages that we _bake into_ the CI test images because those same images are used for provisioning developer reservable nodes and (presumably) developers want the DAOS build dependencies already installed.

Having them already in the image also reduces provisioning time as they don't need to be installed with every test run's cluster provision.

This is rather unfortunate though, as it is indeed masking the lack of dependency specification in `daos-client-tests`.

But overall, I agree, that  `daos-client-tests` should be listing all of the DAOS build dependencies required.  I'm just not sure what the correct balance of preventing regressions in that list and continuing to satisfy the above requirements are.  Ultimately that is a different ticket though.